docs/_running_intro.rst

```diff
@@ -1,3 +1,19 @@
+.. note::
+
+    Some features described in this tutorial are currently available only in
+    the development version of Planemo and may not work correctly when installing
+    the latest version from PyPI or bioconda. To install the development versions
+    of Planemo and galaxy-tool-util required, run the following commands:
+
+    ::
+
+        $ # install dev version of planemo
+        $ pip install https://github.com/galaxyproject/planemo/archive/master.tar.gz
+        $ # install dev version of galaxy-tool-util
+        $ git clone https://github.com/galaxyproject/galaxy.git
+        $ pip install -e galaxy/packages/tool_util
```

Comment:
I published the latest release to PyPI if that helps at all. https://pypi.org/project/galaxy-tool-util/21.1.0.dev4/

Reply:
Yes it does, thanks a lot! You can go ahead and merge this if you like.